Gemfile

```diff
@@ -6,6 +6,7 @@ gem "rails", "~> 8.0.2"
 gem "propshaft"
 # Use sqlite3 as the database for Active Record
 gem "sqlite3", ">= 2.1"
```

Comment:
we can delete sqlite3 because we’re using pg!

Reply:
deleted since i merged in main!